portfolio/src/main/webapp/blogStyle.css

```diff
@@ -49,3 +49,13 @@ input[type=submit] {
   transition: all 0.5s;
   cursor: pointer;
 }
+
+#post-container {
+  padding: 20px;
+  color: #717171;
+  line-height: 25px;
+}
+
+.writePost {
```

Comment:
Class names are hyphenated according to Google CSS style guide, like post-container above.